plugins/node/opentelemetry-instrumentation-undici/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTelemetry Undici/fetch Instrumentation for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides automatic instrumentation for [`undici`](https://undici.nodejs.org/) and [`fetch`](https://nodejs.org/docs/latest/api/globals.html#fetch).
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-undici
+```
+
+## Usage
+
+OpenTelemetry Undici/fetch Instrumentation allows the user to automatically collect trace data and export them to their backend of choice, to give observability to distributed systems.
+
+To load a specific instrumentation (Undici in this case), specify it in the Node Tracer's configuration.
+
+```js
+const { UndiciInstrumentation } = require('@opentelemetry/instrumentation-undici');
+const {
+  ConsoleSpanExporter,
+  NodeTracerProvider,
+  SimpleSpanProcessor,
+} = require('@opentelemetry/sdk-trace-node');
+const { registerInstrumentations } = require('@opentelemetry/instrumentation');
+
+const provider = new NodeTracerProvider();
+
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [new UndiciInstrumentation()],
+});
+
+```
+
+<!-- TODO: -->
+See [examples/http](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/fetch) for a short example.
```

Comment:
I tihnk it may be good to remove the TODO, then add it back later when it's done :slightly_smiling_face: